Lib/distutils/tests/test_check.py

```diff
@@ -1,4 +1,6 @@
 """Tests for distutils.command.check."""
+import os
+import os.path
```

Comment:
Minor: `os` always contains `os.path`

Reply:
done!

Reply:
Ah I meant the opposite: `import os` is enough to get access to os.path functions!